tests/slow/src/test/scala/tests/gradle/GradleLspSuite.scala

```diff
@@ -227,7 +227,7 @@ class GradleLspSuite extends BaseImportSuite("gradle-import") {
         |""".stripMargin
   }
 
-  test("different-scala") {
+  test("different-scala".flaky) {
```

Comment:
is possible to link to an example failure? Would good when trying to debug the issue.

Reply:
We store the full failures along with structured stack traces in Google Cloud. I don't think it's necessary to add comments in the code with example errors, I would prefer to write a tool that allows you to query the data. The data is publicly accessible for read operations, I just need to figure out how to set it up.

Reply:
I opened https://github.com/scalameta/munit/issues/33 as a followup